src/main/java/com/corndel/cashregister/Level5.java

```diff
@@ -27,8 +33,28 @@ public class Level5 {
    * Level1.addItem(...) and so on.
    */
   public static List<Item> transaction(int cost, List<Item> paid, List<Item> drawer) {
-    // TODO
+    // Adds paid money into drawer
+    for (Item item : paid) {
+      for (int i = 0; i < item.quantity; i++) {
```

Comment:
Great work using the methods you had already written in the previous challenges!